modules/manage/partials/tiered-storage.adoc

```diff
@@ -1063,7 +1063,14 @@ rpk topic describe-storage <topic_name> --print-all
 
 See the xref:reference:rpk/rpk-topic/rpk-topic-describe-storage.adoc[reference] for a list of flags you can use to filter the command output.
 
-TIP: A constant stream of data is necessary to build up the log segments and roll them into object storage. To see new log segments faster, you can edit the `segment.bytes` topic-level property. Or, you can edit the xref:reference:properties/object-storage-properties.adoc#cloud_storage_segment_max_upload_interval_sec[`cloud_storage_segment_max_upload_interval_sec`] property, which sets the frequency with which new segments are generated in Tiered Storage, even if the local segment has not exceeded the `segment.bytes` size or the `segment.ms` age.
+[TIP]
+====
+A constant stream of data is necessary to build up the log segments and roll them into object storage. This upload process is asynchronous. You can monitor its status with the following metric:
+
+* xref:reference:internal-metrics-reference.adoc#vectorized_ntp_archiver_pending[`/metrics/vectorized_ntp_archiver_pending`]
```

Comment:
```suggestion
A constant stream of data is necessary to build up the log segments and roll them into object storage. This upload process is asynchronous. You can monitor its status with the following metric: xref:reference:internal-metrics-reference.adoc#vectorized_ntp_archiver_pending[`/metrics/vectorized_ntp_archiver_pending`]
```
@kbatuigas since there's just one metric, I wouldn't put it in a list. Similarly, at the [top of this page](https://deploy-preview-706--redpanda-docs-preview.netlify.app/current/manage/tiered-storage/) is an Important note about upgrades that appears in a list. Can you remove that, so it's just regular text (not list) in the admonition?